docs/content/preview/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -140,100 +128,266 @@ You install YugabyteDB Anywhere on a Kubernetes cluster as follows:
+- For Azure, see [AKS docs](https://docs.microsoft.com/en-us/azure/aks/internal-lb).
+- For AWS, see [EKS docs](https://docs.aws.amazon.com/eks/latest/userguide/load-balancing.html).
 
-```sh
-helm install yw-test yugabytedb/yugaware/ -n yb-platform \
---version 2.15.2 --set yugaware.storageClass=yb-storage \
---set "zoneAffinity={us-west1-a,us-west1-b}"
+For example, for a GKE cluster, you would add following lines to your values file:
+
+```yaml
+# yba-values.yaml
+yugaware:
+  service:
+    # other values…
+    annontations:
+      networking.gke.io/load-balancer-type: "Internal"
 ```
 
-### toleration
+#### TODO Set the DNS name
 
-Kubernetes nodes could have `taints` that repel pods from being placed on it. Only pods with a `toleration` for the same `taint` are permitted. For more information, see [Taints and Tolerations](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/).
+TODO: seems like this is not needed even if using Domain => LB IP setup. It just works fine. We probably need a section for CORS.
 
-For example, if some of the nodes in your Kubernetes cluster are earmarked for experimentation and have a taint `dedicated=experimental:NoSchedule`, only pods with the matching toleration will be allowed; other pods will be prevented from being placed on these nodes.
+### Configure TLS
 
-```sh
-helm install yw-test yugabytedb/yugaware/ -n yb-platform \
---version 2.15.2 --set yugaware.storageClass=yb-storage \
---values=/tmp/overrides.yaml
+YugabyteDB Anywhere can be configured to use TLS, the following customization options are available:
+
+#### Enabled TLS
+
+Add the following lines to your values files to enable TLS. The Helm chart takes care of creating a self signed certificate for you.
+
+```yaml
+# yba-values.yaml
+tls:
+  enabled: true
+```
+
+#### Use custom TLS certificate
+
+You can use custom TLS certificate instead of using the default self signed certificate. Set the value of `certificate` and `key` to the base64 encoded string value of the certificate and the key.
+
+```yaml
+# yba-values.yaml
+tls:
+  enabled: true
+  certificate: "LS0tLS1CRUdJTiBDRVJUSUZJQ..."
+  key: "LS0tLS1CRUdJTiBQUklWQVRFIEtFWS0t..."
+```
+
+#### Change TLS versions
+
+When using TLS with YugabyteDB Anywhere, you can change the supported TLS versions. The value is passed to Nginx frontend as [ssl_protocols](https://nginx.org/r/ssl_protocols) operational directive.
+
+```yaml
+# yba-values.yaml
+tls:
+  enabled: true
+  # other values…
+  sslProtocols: "TLSv1.2 TLSv1.3"
+```
+
+### Control placement of YugabyteDB Anywhere pods
+
+The Helm chart allows you to control the placement of the pods when installing YugabyteDB Anywhere in your Kubernetes cluster via `nodeSelector`, `zoneAffinity` and `toleration`. When you are using these constraints, make sure the storage class is setup according to the [Storage class considerations](#storage-class-considerations) section.
```

Comment:
minor: link to k8s docs like https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/